File: VB.NET/String/FormatNumber.vb
Imports System
Imports System.Globalization
				
Public Module Module1
	Public Sub Main()
		Dim ptbr = new CultureInfo("pt-br")
		Console.WriteLine("{0}", 123.ToString("N2", ptbr))
		Console.WriteLine("{0}", 123.123.ToString("N2", ptbr))
		Console.WriteLine("{0}", 123123132112321321.123d.ToString("N2", ptbr))
		Console.WriteLine("{0}", 0.123.ToString("N2", ptbr))
		Console.WriteLine("{0}", 0.158123.ToString("N2", ptbr))
	End Sub
End Module

'https://pt.stackoverflow.com/q/271855/101

File: CSharp/Console/FormatNumber.cs
using static System.Console;
using static System.Math;
using System.Globalization;

namespace Uri_CSharp {
    public class URI {
        public static void Main(string[] args) {
			if (double.TryParse(ReadLine(), out var raio)) {
				double area = Pow(raio, 2) * 3.14159;
				WriteLine(area.ToString("F4", new CultureInfo("pt-BR")));
			}
		}
    }
}

//https://pt.stackoverflow.com/q/141112/101
